### DISCUSSION TITLE
Question about p2p_S1_TOPS_Frame.csh

### DISCUSSION THREAD

**Discussion by RenzoAnc** [gmtsar/gmtsar#705] [Q&A] [answered]
Hi everyone. I'd like to ask you a question about the p2p_S1_TOPS.csh. I know it allows to form an interferogram of a certain frame of the whole image. But, it's possible to form a interferogram of a certain frame removing certain lines of  the general script (p2p_S1_TOPS_Frame.csh)?.
I mean, if I eliminate the lines of p2p_S1_TOPS_Frame.csh which are referred to the other frames:
![I1](https://user-images.githubusercontent.com/105986961/228896025-f8c25db8-2ff2-4995-9bd7-86d39be9a067.png)
I could form an interferogram of a certain frame  (the image above applies to F2 frame). I applied this script to form interferograms of a certain frame, but I don't know if it's similar to p2p_S1_TOPS.csh.
Script: [script.txt](https://github.com/gmtsar/gmtsar/files/11113544/script.txt)

Regards

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
So you would like to do a interferogram with only two subswaths? This is an interesting question and I didn't thought about that for a pair of interferograms. We definitely could do that with batch processing. You could give it a try by removing other subswaths codes.

Or if you only want to do one subswath, then go to that subswath and run p2p_processing.csh is simpler.

**Reply by RenzoAnc** [reply]
Thank you for your reply, professor @Xiaohua-Eric-Xu . Actually, I did it for an F2 frame (subswath) using a pair of images, executing the script I attached before (script.txt). I could form an interferogram, displacement map and others for this subswath without any problem. I didn't try with two frames (and merge to a unique interferogram) because my AOI is small, but I'll do it in the future.

Based on your answer, I think the processing I did is correct because the script I created (script.txt) to form an interferogram for a subswath is essentially based on this line (p2p_processing.csh). Thank you, professor Xiaohua.
